core/primer.md

```diff
@@ -2,6 +2,185 @@
 
 <!-- no verify-specs -->
 
+## Abstract
+
+This non-normative document provides an overview of the xRegistry specification.
+It is meant to complement the xRegistry specification to provide additional
+background and insight into the history and design decisions made during the
+development of the specification. This allows the specification itself to focus
+on the normative technical details.
+
+## Table of Contents
+
+- [History](#history)
+- [Motivation](#motivation)
+- [Design Goals](#design-goals)
+- [Representations](#representations)
+- [Possible Use Cases](#possible-use-cases)
+- [Design decisions or topics of
+  interest](#design-decisions-or-topics-of-interest)
+
+## History
+
+The CNCF Serverless Working group was originally created by the CNCF's Technical
+Oversight Committee to investigate Serverless Technology and to recommend some
+possible next steps for some CNCF related activities in this space. After
+creating the common event format [CloudEvents](https://github.com/cloudevents)
+as a foundation for an interoperable ecosystem for event-driven applications the
+next step was to create a metadata model for declaring CloudEvents, their
+payloads and associating those declarations with application endpoints. As a
+result, the xRegistry (extensible registry) specification was created.
+
+xRegistry was initially part of CloudEvents, called "CloudEvents Discovery" but
+later moved into it's own repository and CNCF project.
+
+## Motivation
+
+While CloudEvents are harmonizing different event structures across event
+providers and increase interoperability between different brokers and their
+protocols, they do not address:
+
+- **Discovery:** Where to produce/consume events? What endpoints exist?
+- **Validation:** How to validate event structures?
+- **Versioning:** How to version events – and see which versions exist?
+- **Extension Discovery:** How to identify which events using which specific
+  extensions?
+
+Schema registries can provide an answer to these questions, but just like event
+brokers, there are multiple registries, such as the Confluent Schema Registry,
+the Azure Event Hubs Registry or the Apicurio Registry. When an event travels
+through multiple brokers, its schema has to be introduced to the broker's
+registry and clients later have to deal with the implementation differences
+between different registries, for example when trying to validate an event
+structure. This hinders the interoperability of event brokers CloudEvents had in
+mind. xRegistry therefore comes in with a similar motivation to CloudEvents:
+Harmonize another piece of technology in the messaging / eventing ecosystem to
+increase interoperability and decouple event handling from broker products and
+protocols.
+
+Even though xRegistry was built with eventing in mind, the concept of this
+registry specification is far more powerful. Take a look at the [Possible Use
```

Comment:
I'd give a hint here about how it goes beyond eventing... perhaps:
..is far more powerful and can be used for the exchange of any type of message.